utils/navigator_gui/navigator_gui/shooter.py

```diff
@@ -0,0 +1,148 @@
+
+        # Add widget to the user interface
+        context.add_widget(self._widget)
+
+    def connect_ui(self):
+        '''
+        Links objects in the shooter GUI to variables in the backend
+        shooter object.
+        '''
+
+        # Shooter status indicator
+        self.shooter_status_frame = self._widget.findChild(QtGui.QFrame, "shooter_status_frame")
+        self.shooter_status_message = self._widget.findChild(QtGui.QLabel, "shooter_status_message")
+
+        # Control panel buttons
+        load_button = self._widget.findChild(QtGui.QPushButton, "load_button")
+        load_button.clicked.connect(self.remote.shooter_load)
+        fire_button = self._widget.findChild(QtGui.QPushButton, "fire_button")
+        fire_button.clicked.connect(self.remote.shooter_fire)
+        cancel_button = self._widget.findChild(QtGui.QPushButton, "cancel_button")
+        cancel_button.clicked.connect(self.remote.shooter_cancel)
+        reset_button = self._widget.findChild(QtGui.QPushButton, "reset_button")
+        reset_button.clicked.connect(self.remote.shooter_reset)
+        linear_extend_button = self._widget.findChild(QtGui.QPushButton, "linear_extend_button")
+        linear_extend_button.clicked.connect(self.remote.shooter_linear_extend)
+        linear_retract_button = self._widget.findChild(QtGui.QPushButton, "linear_retract_button")
+        linear_retract_button.clicked.connect(self.remote.shooter_linear_retract)
+        disc_speed_slider = self._widget.findChild(QtGui.QSlider, "disc_speed_slider")
+        disc_speed_slider.valueChanged[int].connect(self.cache_disc_speed_setting)
+        set_disc_speed_button = self._widget.findChild(QtGui.QPushButton, "set_disc_speed_button")
+        set_disc_speed_button.clicked.connect(self.set_disc_speed)
+
+        # Defines the color scheme as QT style sheets
+        self.colors = {
+            "red": "QWidget {background-color:#FF432E;}",
+            "green": "QWidget {background-color:#B1EB00;}",
+            "yellow": "QWidget {background-color:#FDEF14;}"
+        }
+        self.status_colors = {"green": ["Standby", "Loaded", "Fired"],
+                              "yellow": ["Loading", "Firing", "Canceling"],
+                              "red": ["Unknown", "Error", "Canceled"]
+                              }
+
+    def cache_disc_speed_setting(self, speed):
+        '''
+        Caches the shooter acceleration disc speed whenever it is updated on
+        the GUI.
+        '''
+        self.disc_speed_setting = speed
+
+    def set_disc_speed(self):
+        '''
+        Sets the shooter's accelerator disc speed to the cached value.
+        '''
+        self.remote.set_disc_speed(self.disc_speed_setting)
+
+    def update_kill_status(self, alarm):
+        '''
+        Updates the kill status display when there is an update on the kill
+        alarm.
+        '''
+        self.is_killed = not alarm.clear
+
+    def update_shooter_status(self, msg):
```

Comment:
It really doesn't effect the GUI operation either way. Manual mode has been added, but all statuses are assumed to be capitalized (I just noticed yours was lowercase).